app/views/admin/videos/new.html.haml

```diff
@@ -0,0 +1,46 @@
+%section.admin_add_video
+  .container
+    .row
+      .col-md-10.col-md-offset-1
+        = form_for [:admin, @video], html: { class: "form-horizontal" } do |f|
```

Comment:
This is no longer the syntax for a horizontal form. It is now `layout: :horizontal`. You can see it in the docs. Once you fix that you will no longer need to specify the form groups and column widths on the form inputs.